storagenode/piecestore/endpoint_test.go

```diff
@@ -265,16 +265,16 @@ func TestDownload(t *testing.T) {
 		buffer := make([]byte, len(expectedData))
 		n, err := downloader.Read(buffer)
 
-		if tt.err != "" {
+		if len(tt.errs) > 0 {
 		} else {
 			require.NoError(t, err)
 			require.Equal(t, expectedData, buffer[:n])
 		}
 
 		err = downloader.Close()
-		if tt.err != "" {
+		if len(tt.errs) > 0 {
 			require.Error(t, err)
-			require.Contains(t, err.Error(), tt.err)
+			require.True(t, strings.Contains(err.Error(), tt.errs[0]) || strings.Contains(err.Error(), tt.errs[1]), err.Error())
```

Comment:
maybe `require.Contains(t, strings.Join(tt.errs, ""), err.Error())`?

Reply:
I was assuming that err.Error() has some other text, in which case that wouldn't work.